.github/workflows/roll.yaml

```diff
@@ -0,0 +1,21 @@
+name: Roll WebRTC revision
+on:
+  push:
+    branches:
+      - 88-sdk
+
+jobs:
+  roll:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: open-webrtc-toolkit/owt-client-native
+        ref: refs/heads/master
+        fetch-depth: 0
+        token: ${{ secrets.OWTBOT_TOKEN }}
+    - uses: actions/setup-python@v2
+    - run: pip install requests
+    - run: python scripts/roll_webrtc.py --base_branch master --revision $GITHUB_SHA --token $OWTBOT_TOKEN
+      env:
+        OWTBOT_TOKEN: ${{ secrets.OWTBOT_TOKEN }}
```

Comment:
OWT_BOT_TOKEN instead of OWTBOT_TOKEN....

Reply:
Done.